From fee0e35d4e652243bfd8cf4c52e1706e0cc9b88d Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Sat, 16 Oct 2021 19:34:10 +0200
Subject: [PATCH] ath11k: Disable coldboot calibration for IPQ8074

There is a bug with the remoteproc reset after coldboot calibration,
so until that is resolved disabled it to allow using the radio.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 drivers/net/wireless/ath/ath11k/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/wireless/ath/ath11k/core.c
+++ b/drivers/net/wireless/ath/ath11k/core.c
@@ -83,7 +83,7 @@ static const struct ath11k_hw_params ath
 		.supports_shadow_regs = false,
 		.idle_ps = false,
 		.supports_sta_ps = false,
-		.cold_boot_calib = true,
+		.cold_boot_calib = false,
 		.fw_mem_mode = 0,
 		.num_vdevs = 16 + 1,
 		.num_peers = 512,
